internal/pkg/build/build.go

```diff
@@ -257,6 +257,8 @@ func (b *Build) Full(ctx context.Context) error {
 	// clean up build normally
 	defer b.cleanUp()
 
+	oldumask := syscall.Umask(0002)
```

Comment:
Since we have this here, do we still need the umask call on line 79? For `newBuild`?

Reply:
in that function, `NewBundle` is doing some operations that will create a tmpdir etc.... I believe we do want to have the `Umask(0002)` for that, so the bundle dir is created with workable perms even if a crazy umask is in force.